app/kumactl/pkg/client/api_server_client.go

```diff
@@ -1,18 +1,28 @@
 package client
 
 import (
+	"fmt"
 	"net/http"
 	"net/url"
+	"runtime"
 	"time"
 
 	"github.com/pkg/errors"
 
 	config_proto "github.com/kumahq/kuma/pkg/config/app/kumactl/v1alpha1"
 	util_http "github.com/kumahq/kuma/pkg/util/http"
+	kuma_version "github.com/kumahq/kuma/pkg/version"
 )
 
 func ApiServerClient(coordinates *config_proto.ControlPlaneCoordinates_ApiServer, timeout time.Duration) (util_http.Client, error) {
-	headers := make(map[string]string)
+	headers := map[string]string{
+		"User-Agent": fmt.Sprintf("kumactl/%s (%s; %s; %s/%s)",
+			kuma_version.Build.Version,
+			runtime.GOOS,
+			runtime.GOARCH,
```

Comment:
@lahabana ,
Those are the actual running OS and architecture informations.
Would it be more appropriate to show https://pkg.go.dev/runtime/debug#BuildSetting instead as they are the actual build data ?

Reply:
I think it's fine Kubernetes uses runtime too: 

https://github.com/kubernetes/kubernetes/blob/4af23c157c3416dbbbd34c94204959c564322a59/staging/src/k8s.io/client-go/rest/config.go#L497-L505